pipeline/dbt/models/intermediate/int__union_adresses__enhanced.sql

```diff
@@ -21,7 +21,7 @@ overriden_adresses AS (
         COALESCE(geocodages.latitude, adresses.latitude)       AS "latitude",
         COALESCE(geocodages.commune, adresses.commune)         AS "commune",
         COALESCE(geocodages.code_postal, adresses.code_postal) AS "code_postal",
-        COALESCE(geocodages.code_insee, adresses.code_insee)   AS "code_insee",
+        COALESCE(geocodages.code_commune, adresses.code_insee) AS "code_insee",
```

Comment:
@vperron donc je peux faire :

```suggestion
        COALESCE(geocodages.code_arrondissement, geocodages.code_commune, adresses.code_insee) AS "code_insee",
```

Reply:
Du coup, pas la peine je pense.

Reply:
ok